### PR TITLE
fix: Fixes #244. Concatenate balances to 2 decimal places on account overview page without rounding

### LINKED ISSUE #244: Inconsistent decimal places uses for Ether account balance
on the "Accounts" page, if I select my new account, it navigates to the account page where it shows I have 0.00 ETH, and if I click that card, it takes me to a "Send Ether" page, where it shows the same card at the top, but it shows I have 0.000000 ETH. is there an opportunity to make the amount of decimal places shown consistent (i.e. 2 dp vs 6 dp).
if the user has less than 1 Szabo (i.e. 0.000001) it may not show their balance even though have a balance.

### PR DESCRIPTION
**Pull request by ltfschoen**
With these changes, given an account with 0.999432 ETH and 1233.8945 TIB.

Previously the account summary page would show: 
  Ether: 1.00
  THIBCoin: 1233.89
Now it shows:
  Ether: 0.99
  THIBCoin: 1233.89

Previously the Send Ether page would show: 0.999432
Now it shows: 0.999432

Previously the Send THIBCoin page would show: 1233.894500
Now it shows: 1233.8945

### CODE REVIEW

**Comment by ltfschoen** [comment]
Note that another approach could be to use BigNumber.js rounding http://mikemcl.github.io/bignumber.js/#constructor-properties

File: packages/fether-ui/src/TokenCard/TokenCard.js [hunk 1]
```diff
@@ -35,7 +43,9 @@ export const TokenCard = ({
       </div>
       <div className='token_balance'>
         {balance ? (
-          <span>{balance.toFixed(decimals)} </span>
+          <span>
+            {concatBalanceByDecimalPlaces(balance.toString(), decimals)}{' '}
```
[the remark this answers is not in the document]
Comment (amaurymartiny) [reply packages/fether-ui/src/TokenCard/TokenCard.js]: ping @ltfschoen Have you had a look at this?

[the remark this answers is not in the document]
Comment (ltfschoen) [reply packages/fether-ui/src/TokenCard/TokenCard.js]: @amaurymartiny i'll take a look now

[the remark this answers is not in the document]
Comment (ltfschoen) [reply packages/fether-ui/src/TokenCard/TokenCard.js]: @amaurymartiny yes, works great!

**Pull request merged**